content/en/docs/tasks/extend-kubernetes/install-service-catalog-using-helm.md

```diff
@@ -3,6 +3,7 @@ title: Install Service Catalog using Helm
 reviewers:
 - chenopis
 content_type: task
+min-kubernetes-server-version: 1.7
```

Comment:
From what I have reviewed, there are a number of changes that do not reflect the full intention of this pull request such as, adding the minimum k8s version, replacing task links with glossary terms, removing blank lines, removing `version check`, removing misc. content, adding an _index page, fixing formatting nits. I am not against folding small issues into a larger change, but sometimes these changes add overhead and would be best created as a separate pull request.
The relevant changes are adjusting `weight` fields to move task files around, creating a new category, moving some single tasks under extend k8s, and updating links.

Wouldn't it be nice if we could add another front matter field to specify an alternative parent directory for this file; if specified use this parent to generate the published file path (instead of constantly moving around files).

Reply:
> adding the minimum k8s version

I can pull that out.

> replacing task links with glossary terms

The side effect is to change a link that redirects into a link that goes to the right place. I can pull those out into their own PR, it will be much _easier_ to merge that change first (or octopus-merge them, but that feels complex).

> Adding `_index` pages

You need this when you make a new section.

> Removing blank lines

I'd prefer to live with that overhead, but I can rebase it out.

Reply:
Hi @sftim . Here are several suggestions to help continue discussion of these changes:
-  Separate the `version check` changes from the task reorganization changes.
-  Separate the miscellaneous clean up changes from the task reorganization changes.
- Try to limit the changes of links to glossary tooltips.
- Push the file front matter `weight` field changes to a new PR.
- Limit this PR to the creation of a new task directory, `manage-resources` and possibly the move of the 
single page tasks to other task categories.

I think doing these things will help focus this pull request and enable reviewers to better understand the changes.